blockchain/blockchain.go

```diff
@@ -1352,19 +1352,27 @@ func (bc *blockchain) createPollGenesisStates(ctx context.Context, ws factory.Wo
 			ws,
 		)
 	}
-	p, ok := bc.protocol(vote.ProtocolID)
-	if !ok {
-		return errors.Errorf("protocol %s is not found", vote.ProtocolID)
-	}
-	vp, ok := p.(*vote.Protocol)
-	if !ok {
-		return errors.Errorf("error when casting vote protocol")
-	}
 	addrs := make([]address.Address, 0)
 	for _, d := range bc.config.Genesis.Delegates {
 		addrs = append(addrs, d.OperatorAddr())
 	}
-	return vp.Initialize(ctx, ws, addrs)
+	for _, addr := range addrs {
```

Comment:
We don't need to initialize following states when voting protocol is deleted.

Do you want to keep it because of breaking tests?

Reply:
+1

Reply:
ok,I'll remove and test it